algorithms/ar-msrt/golang/mergesort.go

```diff
@@ -0,0 +1,76 @@
+// Merge Sort in Golang

```

Comment:
Codacy found an issue: [package comment should be of the form "Package mergesort ..."](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6278215)